tests/src/serverTestSpectrumBuffer-exception.cpp

```diff
@@ -1,7 +1,7 @@
 // SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
 // SPDX-License-Identifier: LGPL-3.0-or-later
 
-#define BOOST_TEST_MODULE serverTestSpectrumBuffer - exception
+#define BOOST_TEST_MODULE (serverTestSpectrumBuffer - exception)
```

Comment:
Are spaces and dashes allowed in the test module name?

Reply:
funny it did not create an error... fixed.